libjolie/src/main/java/jolie/lang/parse/SemanticVerifier.java

```diff
@@ -385,8 +394,19 @@ public void validate()
 			if( executionService != null ) {
 				executionService.program().accept( this );
 				if( configuration.checkForMain && !mainDefined ) {
-					error( program,
-						"Main procedure for service \"" + configuration.executionTarget + "\" is not defined" );
+					// Noticed that sometimes the configuration.executionTarget is null, but the name of the service is
+					// in the executionService
+					// TODO: get the lines of the service missing a main
```

Comment:
remove TODO?

Reply:
Had forgotten to implement that, will commit that fix next time